src/main/groovy/org/ajoberstar/gradle/git/publish/GitPublishPlugin.groovy

```diff
@@ -190,8 +175,9 @@ class GitPublishPlugin implements Plugin<Project> {
           if (!valid) { repo.close() }
           return valid
         }
-    } catch (RepositoryNotFoundException | GrgitException ignored) {
+    } catch (Exception e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The type Exception should not be caught [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.ajoberstar.com/coding_rules#rule_key=grvy%3Aorg.codenarc.rule.exceptions.CatchExceptionRule)